modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc

```diff
@@ -510,6 +510,15 @@ Exception in thread "main" com.couchbase.client.core.error.ParsingFailedExceptio
 
 Not only does it throw a `CouchbaseException`, it also tries to map it to a specific exception type and include extensive contextual information for a better troubleshooting experience.
 
+==== DSL Deprecation
+
+The fluent API feature tried out in SDK 2 was found not to scale well for larger queries, nor for SQL++.
```

Comment:
"tried out" makes it sound like it was experimental, which it wasn't.  I'd just recommend removing those words.

Reply:
+1